ci-operator/step-registry/ibmcloud/provision/vpc/ibmcloud-provision-vpc-commands.sh

```diff
@@ -194,7 +235,7 @@ echo "${vpc_name}" > "${SHARED_DIR}/ibmcloud_vpc_name"
 create_vpc "${CLUSTER_NAME}" "${vpc_name}" "${resource_group}" "${NUMBER_SUBNETS_PAIR_PER_ZONE}"
 
 vpc_info_file="${ARTIFACT_DIR}/vpc_info"
-check_vpc "${vpc_name}" "${vpc_info_file}"
+check_vpc "${vpc_name}" "${vpc_info_file}" $((NUMBER_SUBNETS_PAIR_PER_ZONE*6))
```

Comment:
Why we need to check the number of subnets ? If provision of subnets failed, the creation of vpc would fail directly.

Reply:
easily debugging the created subnets

Reply:
`vpc_info` is already saved in `ARTIFACT_DIR`, you can debug it any time, I do not think we need to change check_vpc any more.

Reply:
too much info in the vpc_info, so I added the subnet list if fail to get the expected subnets number.